controllers/authControll.js

```diff
@@ -19,12 +21,20 @@ const register = async (req, res) => {
   }
   const hashPassword = await bcrypt.hash(password, 10);
   const avatarURL = gravatar.url(email);
+  const verificationToken = nanoid();
 
   const newUser = await User.create({
     ...req.body,
     password: hashPassword,
     avatarURL,
+    verificationToken,
   });
+  const verifyEmail = {
```

Comment:
```suggestion
  const msgOpt = {
```
Zmieniłbym to np na msgOpt lub opt.